client/question/components/LearnerViewDirectiveSpec.js

```diff
@@ -219,6 +219,146 @@ describe('LearnerViewDirective', function() {
     });
   });
 
+  describe("isDocumentHidden", function() {
+    it('should return document[hiddenAttributeName]', function() {
+      expect($scope.isDocumentHidden('sure')).toEqual(document.sure);
```

Comment:
Maybe just use 'hidden' or 'attrname'? Hard to tell from context what 'sure' represents.